ibridges/irodsconnector/data_operations.py

```diff
@@ -85,23 +94,49 @@ def get_collection(self, path: IrodsPath) -> irods.collection.iRODSCollection:
             Instance of the collection with `path`.
 
         """
-        if self.collection_exists(path):
-            return self.session.irods_session.collections.get(path)
+        path = IrodsPath(self.session, path)
```

Comment:
I do not think so. The function takes the whole IrodsPath as a string and checks if the path in its entirety exists on the iRODS server. If so it returns the collection, if not it will raise a CollectionNotFound exception